google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -41,16 +41,17 @@ class Pipeline(object):
 from airflow.contrib.operators.bigquery_table_delete_operator import BigQueryTableDeleteOperator
 from airflow.contrib.operators.bigquery_to_bigquery import BigQueryToBigQueryOperator
 from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
-from datetime import datetime, timedelta
+from datetime import timedelta
```

Comment:
No real reason. I think I was just trying to reduce complexity from the imports and move them down to the invocation sites in cases where the number of invocations are low (2 in this case).